HQSmokeTests/testCases/conftest.py

```diff
@@ -0,0 +1,154 @@
+            settings[name] = os.environ[var]
+    return settings
+
+
+def load_settings():
+    if os.environ.get("CI") == "true":
+        settings = load_settings_from_environment()
+        if not settings:
+            raise RuntimeError(
+                f"Environment variables not set:\n"
+                "  DIMAGIQA_URL\n"
+                "  DIMAGIQA_LOGIN_USERNAME\n"
+                "  DIMAGIQA_LOGIN_PASSWORD\n\n"
+                "See https://docs.github.com/en/actions/reference/encrypted-secrets "
+                "for instructions on how to set them."
+            )
+        return settings
+    path = Path(__file__).parent.parent / "settings.cfg"
+    if not path.exists():
+        raise RuntimeError(
+            f"Not found: {path}\n\n"
+            "Copy settings-sample.cfg to settings.cfg and populate "
+            "it with values for the environment you want to test."
+        )
+    settings = ConfigParser()
+    settings.read(path)
+    return settings["default"]
+
+@pytest.fixture(scope="class")
+def init_driver(request):
+    settings = load_settings()
+    chrome_options = Options()
+    global driver
+    if os.environ.get("CI") == "true":
+        chrome_options.add_argument('--no-sandbox')
+        chrome_options.add_argument('disable-extensions')
+        chrome_options.add_argument('--safebrowsing-disable-download-protection')
+        chrome_options.add_argument('--safebrowsing-disable-extension-blacklist')
+        chrome_options.add_argument('window-size=1920,1080')
+        chrome_options.add_argument('--no-sandbox')
+        chrome_options.add_argument('--start-maximized')
+        chrome_options.add_argument('--headless')
+        chrome_options.add_experimental_option("prefs", {
+            "download.default_directory": str(UserInputsData.download_path),
+            "download.prompt_for_download": False,
+            "download.directory_upgrade": True,
+            "safebrowsing.enabled": True})
+    else:
+        chrome_options.add_argument('--safebrowsing-disable-download-protection')
+        chrome_options.add_argument('--safebrowsing-disable-extension-blacklist')
+        # #
+        # chrome_options.add_argument("--window-size=1920,1080")
+        # chrome_options.add_argument('--start-maximized')
+        # chrome_options.add_argument('--no-sandbox')
+        # chrome_options.add_argument("--disable-extensions")
+        # chrome_options.add_argument('--headless')
+
+        chrome_options.add_experimental_option("prefs", {
+            "download.default_directory": str(UserInputsData.download_path),
+            "download.prompt_for_download": False,
+            "safebrowsing.enabled": True})
+    web_driver = ChromeDriverManager().install()
+
+    driver = webdriver.Chrome(executable_path=web_driver, options=chrome_options)
+    request.cls.driver = driver
+    login = LoginPage(driver, settings["url"])
+    login.login(settings["login_username"], settings["login_password"])
```

Comment:
This should be a separate fixture or part of test setup rather than part of the driver setup